src/org/jitsi/impl/neomedia/transform/rtcp/StatisticsEngine.java

```diff
@@ -971,8 +971,8 @@ private void updateReceivedMediaStreamStats(
                     if (logger.isTraceEnabled())
                     {
                         logger.trace(
-                                "Received estimated bitrate (bps): "
-                                        + remb.getBitrate() + ", dest: "
+                                "remb,stream=" + mediaStream.hashCode() +
```

Comment:
Can we please change remb to remb_received or something like this? I know this sounds stupid, but I have to admin I've had cases where I was looking at the REMBs in the wrong direction...